README.md

```diff
@@ -1,7 +1,8 @@
 # Nightly upload
 
-This is a GitHub Action that uploads nightly builds to the [scientific-python nightly channel][],
-as recommended in [SPEC4 — Using and Creating Nightly Wheels][].
+This is a GitHub Action that uploads (and helps remove) nightly builds to
+the [scientific-python nightly channel][], as recommended in
+[SPEC4 — Using and Creating Nightly Wheels][].
```

Comment:
These are two separate actions meant for very different audiences. Please do not combine them in the same areas of the README. They should be separated into different sections.